doc/conf.py

```diff
@@ -26,11 +27,20 @@
 # Add the 'extensions' directory to sys.path, to enable finding Sphinx
 # extensions within.
 sys.path.insert(0, os.path.join(ZEPHYR_BASE, 'doc', 'extensions'))
-# Also add west, to be able to pull in its API docs.
-sys.path.append(os.path.join(ZEPHYR_BASE, 'scripts', 'meta'))
-# HACK: also add the runners module, to work around some import issues
-# related to west's current packaging.
-sys.path.append(os.path.join(ZEPHYR_BASE, 'scripts', 'meta', 'west'))
+
+try:
+    desc = check_output(['west', 'list', '-f{abspath}', 'west'],
+			stderr=DEVNULL,
+			cwd=os.path.dirname(__file__))
+    west_path = desc.decode(sys.getdefaultencoding()).strip()
+    # Add west, to be able to pull in its API docs.
+    sys.path.append(os.path.join(west_path, 'src'))
+    # HACK: also add the runners module, to work around some import issues
+    # related to west's current packaging.
+    sys.path.append(os.path.join(west_path, 'src', 'west'))
+except CalledProcessError as e:
+    # Do nothing, there will be warnings during the build
```

Comment:
Are you sure about this? We won't be able to build the docs for west's runners; why not fail fast since documentation takes a long time to build especially on slow machines?

Reply:
Yes good point. I've fixed this now.